index.html

```diff
@@ -100,169 +95,319 @@ <h1>Am Alphy glad to see you!</h1>
+                <div class="project-button pro-btn-one">
                     <button>See Project</button>
                 </div>
+                
             </div>
         </section>
         <section class="about">
-            <div class="about-title">
+            <div class="about-title n-about-one">
                 <h3>About <br>Myself</h3>
             </div>
-            <div class="about-description">
+            <div class="about-description n-desc-one">
                 <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
                     of my
                     work and experience! If you like what you see and have a project you need coded, don’t hestiate to
                     contact
                     me.</p>
                 <p class="connect">Let's connect</p>
             </div>
-            <div class="about-social">
-                <ul>
+            <div class="about-social n-social-one">
+                <ul class="about-s-one">
                     <li><a href="#"><img src="./images/github.png"></a></li>
                     <li><a href="#"><img src="./images/linkedin.png"></a></li>
                     <li><a href="#"><img src="./images/hand.png"></a></li>
                     <li><a href="#"><img src="./images/twitter.png"></a></li>
                     <li><a href="#"><img src="./images/m.png"></a></li>
                 </ul>
             </div>
-            <div class="about-resume">
+            <div class="about-resume n-resume-one">
                 <button class="btn-resume">Get my resume</button>
             </div>
+            <div id="new-main-about">
+                <div class="about-title">
+                    <h3>About <br>Myself</h3>
+                </div>
+                <div class="about-description">
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my
+                        work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                        contact
+                        me.</p>
+                    <p class="connect">Let's connect</p>
+                </div>
+                <div class="about-social">
+                    <ul>
+                        <li><a href="#"><img src="./images/github.png"></a></li>
+                        <li><a href="#"><img src="./images/linkedin.png"></a></li>
+                        <li><a href="#"><img src="./images/hand.png"></a></li>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/m.png"></a></li>
+                    </ul>
+                </div>
+                <div class="about-resume">
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+            </div>
```

Comment:
- [ Optional ] Please add some space on top of the JS and HTML icons to match the design 

| Yours  |Figms |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215292742-fab81651-f6a5-41bf-907d-f113a1190027.png) | ![image](https://user-images.githubusercontent.com/74744787/215292764-72bd1bd2-3781-43bf-9e66-b497c2c8c82b.png)  |